src/client/core/xhr-barrier.js

```diff
@@ -0,0 +1,117 @@
+import hammerhead from './deps/hammerhead';
```

Comment:
I thought we name it `autowaits` and move it to `driver` code. There is no reason for this to exist in `core`